### PR TITLE
Add model reload feature

### LINKED ISSUE #22: Add Model reload
Proposed enhancement: trigger model reload. Now it's impossible (?) to reload model. The workaround is to disable and reenable the node. It would be perfect to be able to automatize it.

### PR DESCRIPTION
**Pull request by acejacek**

### CODE REVIEW

File: teachable_machine.html [hunk 1]
```diff
@@ -70,6 +70,8 @@ <h3>Inputs</h3>
         <dl class="message-properties">
             <dt>payload<span class="property-type">image buffer</span></dt>
                 <dd>A binary buffer of an image</dd>
+            <dt>reload<span class="property-type">bool</span></dt>
+                <dd>Set to any value to reload the model. In the same time, ignore image in payload (if any) and do not perform image classification.</dd>
```
Comment (bonastreyair) [inline teachable_machine.html:74]: I would say send a `true` value.
```suggestion
                <dd>Set to true to reload the model. In the same time, ignore image in payload (if any) and do not perform image classification.</dd>         
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-teachable-machine",
-  "version": "1.2.2",
+  "version": "1.2.3",
```
Comment (bonastreyair) [inline package.json:3]: Version 1.3.0 since this adds a new functionality that it is backwards compatible
```suggestion
  "version": "1.3.0",
```

**Review by bonastreyair — CHANGES_REQUESTED** [review]
I like the idea but I see here we could split it into two nodes then:
- an `inference` node that just runs inferences of input images (with threshold configs)
- a separate `configuration` or `model` node, where model load, reload related stuff can be statically and dynamically defined

Let me know what you think about it. (I might accept this PR anyway once I tested it, since it is an improvement over current version!)

Remember that doing this change, multiple models should still be able to be loaded with different nodes so this new `inference` node should be able to select which model configuration node to use.

**Comment by acejacek** [comment]
You have an interesting point with this split: having a separate nodes for config and runtime can solve the conflict I have currently with simultaneous processing of image and model reload (due to async processing). Worth to investigate.

**Comment by bonastreyair** [comment]
I'm going to merge it to dev for testing development and refinement and then publish to main

**Review by bonastreyair — APPROVED** [review]

**Comment by bonastreyair** [comment]
Thanks a lot! Check out release `1.3.0` is out already in npm and Node-RED!

**Pull request merged**